release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -568,6 +568,80 @@ For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-
 [id="ocp-4-11-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+==== Monitoring stack components and dependencies
+
+Updates to versions of monitoring stack components and dependencies include the following:
+
+* Alertmanager to 0.24.0
+* kube-state-metrics to 2.5.0
+* node-exporter to 1.3.1
+* prom-label-proxy to 0.4.0
+* Prometheus to 2.36.2
+* Prometheus adapter to 0.9.1
+* Prometheus operator to 0.57.0
+* Thanos to 0.26.0
+
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *New*
+** Added the `KubePersistentVolumeInodesFillingUp` alert, which functions similarly to the existing `KubePersistentVolumeFillingUp` alert but is applied to inodes rather than volume space.
+** Added the `PrometheusScrapeBodySizeLimitHit` alert to detect targets hitting the body size limit.
+** Added the `PrometheusScrapeSampleLimitHit` alert to detect targets hitting the sample limit.
+
+* *Changed*
+** Fixed the `KubeDaemonSetRolloutStuck` alert to use the updated metric `kube_daemonset_status_updated_number_scheduled` from `kube-state-metrics`.
+** Replaced the `KubeJobCompletion` alert with `KubeJobNotCompleted`. The new `KubeJobNotCompleted` alert avoids false positives when an earlier job failed but the most recent job succeeded.
+** Updated the `NodeNetworkInterfaceFlapping` alert to exclude `tunbr` interfaces from the alert expression.
+** Updated the `PrometheusOperatorWatchErrors` alert to reduce the time interval from ten minutes to five minutes, to enable faster recovery of the alert, and to reduce false positives.
+
+==== Enable alert routing for user workload monitoring
+A cluster administrator can now enable alert routing for user workload monitoring so that developers and other users can configure custom alerts and alert routing for their user-defined projects.
+
+==== Enable a dedicated Alertmanager instance for user-defined alerts
+You can optionally enable a separate instance of Alertmanager dedicated to sending alerts only for user-defined projects.
+This feature can help reduce the load on the default platform Alertmanager instance and can better separate user-defined alerts from default platform alerts.
+
+==== Use additional authentication settings for remote write configuration
+You can now use the following authentication methods to access a remote write endpoint: AWS Signature Version 4, authorization, and OAuth 2.0. 
```

Comment:
```suggestion
You can now use the following authentication methods to access a remote write endpoint: AWS Signature Version 4, Authorization (custom Authorization header), and OAuth 2.0. 
```
Adding a bit of detail.

Reply:
Changed to Simon's suggestion (fewer words)